server/tests/api_v5/anonymous_account/test_account_send_email_validation.py

```diff
@@ -0,0 +1,45 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import pytest
+
+from parsec._parsec import anonymous_account_cmds
+from tests.common import Backend, HttpCommonErrorsTester
+from tests.common.client import AccountRpcClient
+
+
+async def test_anonymous_account_account_send_email_validation_token_ok(
+    account: AccountRpcClient, backend: Backend, with_postgresql: bool
+) -> None:
+    if with_postgresql:
+        pytest.xfail("TODO: postgre not implemented yet")
```

Comment:
You can avoid these `if`s with some handy fixtures from `conftest.py`:

https://github.com/Scille/parsec-cloud/blob/0327720b970b9217bc1e9e1ea8c366022776236b/server/tests/conftest.py#L251-L260

Reply:
There are a few other places where this is the case, see #10200